vimeo/VimeoUpload
# Upload analytics

# Ticket(s)

https://vimean.atlassian.net/browse/VIM-3226
# Ticket Summary

Implement analytics for the new upload system with some improvements over the existing schema (see analytics FRD linked from https://vimean.atlassian.net/browse/VIM-3190)
# Implementation Summary
- UploadAnalyticsController is a new object that handles sending analytics events for one upload in progress.  It acts as a simple state machine, using tracking variables to ensure that events are sent only once and in the correct order.
# How to test

See verbose analytics output in the Xcode console and use it to verify that Attempt, Start, Success, Failure, and Cancel upload events are sent correctly.
